base/task.jl

```diff
@@ -607,7 +607,7 @@ function wait()
     W = Workqueues[Threads.threadid()]
     reftask = poptaskref(W)
     result = try_yieldto(ensure_rescheduled, reftask)
-    process_events()
+    Sys.isjsvm() || process_events()
```

Comment:
I presume this is because JS is doing the event processing for us?

Reply:
yes